src/amp-user-notification(with_backend_service).html

```diff
@@ -0,0 +1,80 @@
+<!--
```

Comment:
OK - let's rename then both samples (just rename the files), e.g.:

* 'amp-user-notification_with_local_storage'
* 'amp-user-notification_with_server_endpoint'